docs/03_configuration/README.md

```diff
@@ -2,32 +2,32 @@
 
 After installing EasyBuild you should **configure EasyBuild to your preferences**.
 
-Although EasyBuild should work fine out-of-the-box assuming that you are using Lmod
-as modules tool (if not, see <a href="#modules-tool-module-syntax">here</a>),
-you will probably not be happy with the default configuration.
+EasyBuild should work fine out-of-the-box if you use Lmod
+as your modules tool. If you are not using Lmod, please see <a href="#modules-tool-module-syntax">here</a> for more information. 
+Either way you should go through the default configuration and expect to make local adaptations.
```

Comment:
This should be worded differently imho.

Other than the modules tool, the default EasyBuild configuration will work out of the box, but it'll use `$HOME/.local/easybuild`, which I doubt many people will like.

Reply:
Yeah, stronger language required. Check.